src/main/java/org/opensearch/agent/tools/SearchAnomalyResultsTool.java

```diff
@@ -39,7 +39,8 @@
 @ToolAnnotation(SearchAnomalyResultsTool.TYPE)
 public class SearchAnomalyResultsTool implements Tool {
     public static final String TYPE = "SearchAnomalyResultsTool";
-    private static final String DEFAULT_DESCRIPTION = "Use this tool to search anomaly results.";
+    private static final String DEFAULT_DESCRIPTION =
+        "This is a tool that searches anomaly results. It takes 9 arguments named detectorId which defines the detector ID to filter for (default is null), and realtime which defines whether the anomaly is real time, and anomalyGradeThreshold which defines the threshold for anomaly grade (a number between 0 and 1 that indicates how anomalous a data point is) (default is 0), and dataStartTime which defines the start time of the anomaly query (default is null), and dataEndTime which defines the end time of the anomaly query (default is null), and sortOrder which defines the order of the results (options are asc or desc, and default is desc), and sortString which which defines how to sort the results (default is data_start_time), and size which defines the size of the request to be returned (default is 20), and startIndex which defines the index to start from (default is 0).The tool returns a list of anomaly results.";
```

Comment:
Same thing as previous comment - maybe something like "The tool returns a list of anomaly results, and the total number of anomaly results"